rdkit/Chem/fmcs/fmcs.py

```diff
@@ -743,20 +741,19 @@ def remove_unknown_bondtypes(typed_mol, supported_canonical_bondtypes):
 
 
 def find_upper_fragment_size_limits(rdmol, atoms):
-  max_num_atoms = max_twice_num_bonds = 0
+  max_num_atoms, max_twice_num_bonds = 0, 0
```

Comment:
No. I'm not a fan of multiple assignments, but converting this to tuple unpacking makes no sense.